Build/local_docker_run.ps1

```diff
@@ -0,0 +1,113 @@
+# Script taken from https://github.com/zifro-playground/car-controller
+# Copyright Zifro © 2019
+
+# Opens the current folder with the docker image
+
+# THIS SCRIPT IS MEANT TO BE USED FOR DEVELOPMENT PURPOSES
+# DO NOT USE IN PRODUCTION
+
+param(
+    [string]
+    $VolumeSource = "/c/dev/Newtonsoft.Json-for-Unity.Converters",
+
+    # Unity license.ulf
+    [string]
+    $UnityLicenseOverride,
+
+    [string]
+    $DockerImage = "applejag/newtonsoft.json-for-unity.converters.package-builder",
+
+    [ValidateSet("2018.4.14f1", "2019.2.11f1")]
+    [string]
+    $UnityVersion = "2019.2.11f1",
+
+    [int]
+    [ValidateRange(1, [int]::MaxValue)]
+    $DockerImageVersion = 1,
+
+    [string]
+    $DockerImageOverride = ""
+)
+
+$ErrorActionPreference = "Stop"
+
+if (-not [string]::IsNullOrEmpty($DockerImageOverride)) {
+    $DockerImage = $DockerImageOverride
+} elseif ($DockerImage.IndexOf(':') -eq -1) {
+    $DockerImage = "${DockerImage}:v$DockerImageVersion-$UnityVersion"
+}
+
+$UnityLicenseULF = if (-not [string]::IsNullOrEmpty($UnityLicenseOverride)) {
+    Resolve-Path $UnityLicenseOverride
+} else {
+    Join-Path "$PSScriptRoot" "Unity_v$UnityVersion.ulf"
+}
+
+$Command = '/bin/bash'
+$Args = @(
+    , "-e", "TEST_PLATFORM=linux"
+    , "-e", "WORKDIR=/root/repo"
+    , "-e", "SCRIPTS=/root/repo/ci/scripts"
+    , "-v", "${VolumeSource}:/root/repo"
+)
+
+if (Test-Path $UnityLicenseULF)
+{
+    Write-Output "Using Unity license $UnityLicenseULF"
+    Write-Output "Using Docker image $DockerImage"
+    Write-Output "Using volume $VolumeSource at /root/repo"
+    
+    $UnityLicenseContent = Get-Content -Path $UnityLicenseULF -Raw
+    $UnityLicenseBytes = [System.Text.Encoding]::UTF8.GetBytes($UnityLicenseContent)
+    $UnityLicenseB64 = [Convert]::ToBase64String($UnityLicenseBytes)
+
+    $Args += @(
+        , "-e", "UNITY_LICENSE_CONTENT_B64=$UnityLicenseB64"
+    )
+    
+    Write-Output ""
+
+    docker run -it --rm $Args $DockerImage /bin/bash
+}
+else
+{
+    Write-Output "No Unity license at $UnityLicenseULF, going for fetching login instead"
+    Write-Output "Using Docker image $DockerImage"
+    Write-Output "Using volume $VolumeSource at /root/repo"
+    
+    $UnityID = Get-Credential -Message "Enter UnityID login"
+    $UnityUserName = $UnityID.UserName
+    $UnityPassword = $UnityID.GetNetworkCredential().Password
+    $Args += @(
+        , "-e", "UNITY_USERNAME=$UnityUserName"
+        , "-e", "UNITY_PASSWORD=$UnityPassword"
+    )
+    $Command = @"
+    xvfb-run -as '-screen 0 640x480x24' \
+        /opt/Unity/Editor/Unity \
+        -logFile /dev/stdout \
+        -batchmode \
+        -username '$UnityUserName' -password '$UnityPassword'
+    echo 
+"@
+
+    Write-Host ""
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [File 'local_docker_run.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5097765)